python/neuroglancer/pipeline/storage.py

```diff
@@ -336,16 +362,22 @@ def delete_file(self, file_path):
         k.key = self.get_path_to_file(file_path)
         self._bucket.delete_key(k)
 
-    def list_files(self, prefix):
+    def list_files(self, prefix, flat=False):
         """
-        if there is no trailing slice we are looking for files with that prefix
+        List the files in the layer with the given prefix. 
+
+        flat means only generate one level of a directory,
+        while non-flat means generate all file paths with that 
+        prefix.
         """
-        from tqdm import tqdm
+
         layer_path = self.get_path_to_file("")        
         path = os.path.join(layer_path, prefix)
         for blob in self._bucket.list(prefix=path):
-            filename =  os.path.basename(prefix) + blob.name[len(path):]
-            if '/' not in filename:
+            filename = blob.name.replace(layer_path + '/', '')
```

Comment:
hrmm wouldn't trimming the text be faster than replacing? or does that not work for these functions?